packages/core/src/types/EventTypes.ts

```diff
@@ -223,8 +223,7 @@ type ImageSpacingCalibratedEventDetail = {
   viewportId: string;
   renderingEngineId: string;
   imageId: string;
-  rowScale: number;
-  columnScale: number;
+  scale?: number;
```

Comment:
why are we getting rid of scale? is it the case that we can't have different scale in rows vs columns?

Reply:
Because the rescale for measurements is fundamentally broken in ways that corrupt the underlying data depending on exactly what is and isn't applied at any given time.  I don't see any way to fix that in a way that works.
IF we want to have aspect ratio measurement changes, then we need to apply it to each measurement type calculations.